crates/extensions/tedge_mqtt_bridge/tests/bridge.rs

```diff
@@ -398,7 +467,7 @@ impl Proxy {
                         };
 
                         write_socket.shutdown().await.unwrap();
-                        write_conn.shutdown().await.unwrap();
+                        let _ = write_conn.shutdown().await; //.await.unwrap();
```

Comment:
Why accepting any return value? with `.await.unwrap()` looks good enough although it's commented out.

Reply:
We want to shut the connection down cleanly, but it may have already closed. If it is already closed, we should just ignore the error we get.